learning_journal/models.py

```diff
@@ -59,6 +59,21 @@ def by_id(cls, id, session=None):
         return entry
 
 
+class User(Base):
+    __tablename__ = 'users'
+    id = Column(Integer, primary_key=True)
+    username = Column(Unicode(255), nullable=False, unique=True, index=True)
+    password = Column(Unicode, nullable=False)
+
+    @classmethod
+    def user(cls, name, session=None):
+        if session is None:
+            session = DBSession
+        all_records = session.query(cls)
+        entry = all_records.get(name)
+        return entry
```

Comment:
I like that you are being very explicit in your work here.  At some point you'll probably want to switch to the more terse chained syntax, like so:

```python
def user(cls, name, session=None):
    #...
    return session.query(cls).get(name)
```
Don't make that change until *you personally* can look at that line of code and see each of the three lines you wrote.  At that point you'll be ready.